sdk/src/network/public/fetcher/fetcher-utils.ts

```diff
@@ -0,0 +1,48 @@
+import { Address, AddressUtil } from "@orca-so/common-sdk";
+import { Connection } from "@solana/web3.js";
+import invariant from "tiny-invariant";
+import { AccountName, WHIRLPOOL_CODER, WhirlpoolData, getAccountSize } from "../../../types/public";
+import { ParsableWhirlpool } from "../parsing";
+
+/**
+ * Retrieve a list of cached whirlpool addresses and accounts filtered by the given params using
+ * getProgramAccounts.
+ * @category Fetcher
+ *
+ * @param connection The connection to use to fetch accounts
+ * @param programId The Whirlpool program to search Whirlpool accounts for
+ * @param configId The {@link WhirlpoolConfig} account program address to filter by
+ * @returns tuple of whirlpool addresses and accounts
+ */
+export async function getAllWhirlpoolAccounts({
```

Comment:
I don't have strong opinion about it, but I think "ByConfig" could be appended to the name.
(To make sure that the name can be used with more ByX)
WDYT ?